### PR TITLE
clean up + fix issue that sometime cant unlock wallet

### PR DESCRIPTION
**Pull request by djhahe**
### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: client/src/components/web-extension/Common/LoginModal/LoginModalForm.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const LoginModalConfirm = ({isOpen, onLoginSuccess, title = ''}) => {
 					<div className="cd_we_create-wallet-layout--body">
 						<AuthLogin
 							onLoginSuccess={onLoginSuccess}
-							passwordLabel="Please login again to continue using Casper Wallet."
+							passwordLabel="Please login again to continue using the Wallet."
```
Comment (duc-q) [inline client/src/components/web-extension/Common/LoginModal/LoginModalForm.jsx:18]: Minor: I notice some removal of "Casper" word in text/label. Can you explain more on this action? Does it affect Casper Wallet brand?

File: client/src/services/ServiceWorker/MessageChannel.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default class MessageChannel {
 		if (request.destination === this.source && request.source === this.destination) {
 			let promise;
 			try {
-				if (this.logMessages) {
+				if (CONSTANTS.DEBUG_ENV) {
```
Comment (duc-q) [inline client/src/services/ServiceWorker/MessageChannel.js:19]: Nice one

File: client/src/app/web-extension/App.js [hunk 3]
```diff
@@ -27,6 +28,13 @@ const getRoutes = (routes) => {
 const App = () => {
 	const { mainRoutes, innerRoutes, outerRoutes } = routeConfig;
 
+	useEffect(() => {
+		setPopupOpenState(true);
```
Comment (duc-q) [inline client/src/app/web-extension/App.js:32]: This is something I mentioned with bad variable naming, `Popup` is a generic name, it's not clear enough to tell what `popup` specifically does.

For quick resolve, I recommend:
`import { setPopupOpenState as setAuthModalOpenState } from '@cd/components/hooks/useServiceWorker';`

**Review by duc-q — APPROVED** [review]
Just minor code change, otherwise look good to me

File: client/src/app/web-extension/App.js (hunk 3 above)
Comment (kien32co) [reply client/src/app/web-extension/App.js:32]: It's not authModal a @duc-q , the popup here mean the extension popup  . Let me update the name

Comment (djhahe) [reply client/src/app/web-extension/App.js:32]: It's not authModal a @duc-q , the popup here mean the extension popup . Let me update the name

File: client/src/components/web-extension/Common/LoginModal/LoginModalForm.jsx (hunk 1 above)
Comment (djhahe) [reply client/src/components/web-extension/Common/LoginModal/LoginModalForm.jsx:18]: We're not Casper wallet @duc-q , our name is name is CasperDash and there is another Casper wallet in the market.

**Pull request merged**